### DISCUSSION TITLE
Recommended way to monitor `zrepl` on a home machine?

### DISCUSSION THREAD

**Discussion by bakhtiyarneyman** [zrepl/zrepl#702] [Q&A] [answered]
I'm using NixOS on my home machine, and `zrepl` service is logging into the journal by default. But the priority is  `info`, even on errors (they are merely prepended with the string `[ERRO]`). It's easy to miss. What's the recommended way for me to be alerted about this? I would like something that both writes errors to the journal at an error priority (because I have a process that translates these to `notify-send` calls), as well as sends an email.

**Comment by kpfleming** [comment, marked as answer]
`level: info` in the zrepl configuration does not mean all messages are logged at the `info` level. It means that messages with level `info` or above will be logged.

I have this logging configuration:
```
global:
  logging:
    - type: syslog
      format: human
      level: info
```

With this in place, zrepl errors and warnings are properly logged (into the journal) with their levels.

I then use the [journal-brief](https://github.com/twaugh/journal-brief) tool to send out emails that include any messages at levels above `info`.
